### DISCUSSION TITLE
iOS: Often crashes when loading beatmap on mania and taiko

### DISCUSSION THREAD

**Discussion by littlemanthe1** [ppy/osu#17753] [Q&A]
### Type

Performance

### Bug description

When loading into any beatmap in taiko or mania, Osu crashes unexpectedly. The audio cuts in and out several times before this.

From testing, osu and catch seem to load normally without error.

### Version

2022.409.0 (also experienced the same crashes in 2022.405.0 and 2022.327.0 builds)

### Logs

I am sadly unable to retrieve logs, as I do not have a functioning computer to access the iOS log files at this time.

**Comment by peppy** [comment]
Probably running out of memory. What device, what beatmap?

**Comment by littlemanthe1** [comment]
All beatmaps on taiko and mania, running on an iPhone XR, with about 30gb of storage space free

**Comment by peppy** [comment]
Unfortunately we can't do much without logs, or knowing a specific beatmap which causes the issue.

**Reply by littlemanthe1** [reply]
I could provide a video and a beatmap that causes the crash, will take only a few minutes to get

**Reply by littlemanthe1** [reply]
> I could provide a video and a beatmap that causes the crash, will take only a few minutes to get

The beatmap that caused this one was https://osu.ppy.sh/beatmapsets/1173144#mania/2446751, though the same crash happens on other beatmaps

I quickly recorded and uploaded the video to YouTube, where it is unlisted: https://youtu.be/hERATujEsv8

**Reply by frenzibyte** [reply]
That's still not enough, logs are required and we can't proceed without that.
